rialto_airflow/harvest/dimensions.py

```diff
@@ -211,4 +212,4 @@ def fill_in(snapshot: Snapshot, jsonl_file: Path) -> Path:
 
     logging.info(f"filled in {count} publications")
 
```

Comment:
It's more important that we return the path to the new JSON file than it is the snapshot that was passed in?